ddl/table.go

```diff
@@ -517,6 +517,45 @@ func onTruncateTable(d *ddlCtx, t *meta.Meta, job *model.Job) (ver int64, _ erro
 		return 0, errors.Wrapf(err, "failed to notify PD the placement rules")
 	}
 
+	tableRuleID := fmt.Sprintf(label.TableIDFormat, label.IDPrefix, job.SchemaName, tblInfo.Name.L)
```

Comment:
Maybe add a global variable switch to enable/disable the feature, `GetLabelRules` is a must need, thus network performance may damage... Placement rules have a cache to avoid unnecessary requests completely.